apps/files/lib/Listener/SyncLivePhotosListener.php

```diff
@@ -37,6 +37,7 @@ class SyncLivePhotosListener implements IEventListener {
 	private array $pendingRenames = [];
 	/** @var Array<int, bool> */
 	private array $pendingDeletion = [];
+	private string $copyTargetPath = '';
```

Comment:
This need to be an array as multiple live photo could be copied at once. When copying a folder, for example. Also, maybe use the same semantic and variable type as the other variables. An array of fileids would be enough I think.

```suggestion
	private string $pendingCopy = [];
```

Reply:
Ah, good point! Changed this now.